### PR TITLE
Allow compilation of the Java wrapper for libjpeg-turbo

### PR DESCRIPTION
**Pull request by agarie**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/jpeg-turbo.rb [hunk 1]
```diff
@@ -30,6 +31,10 @@ def install
     cp Dir["#{Formula["libtool"].opt_share}/libtool/*/config.{guess,sub}"], buildpath
     args = %W[--disable-dependency-tracking --prefix=#{prefix} --with-jpeg8 --mandir=#{man}]
 
+    if build.with? "java"
+      args.push "--with-java"
```
Comment (MikeMcQuaid) [inline Formula/jpeg-turbo.rb]: `args << "--with-java" if build.with? "java"`

Also, does this require Java to be installed? If so, add a `depends_on :java` above.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

File: Formula/jpeg-turbo.rb [hunk 2]
```diff
@@ -22,14 +22,18 @@ class JpegTurbo < Formula
   keg_only "libjpeg-turbo is not linked to prevent conflicts with the standard libjpeg."
 
   option "without-test", "Skip build-time checks (Not Recommended)"
+  option "with-java", "Generate the Java wrapper"
 
   depends_on "libtool" => :build
   depends_on "nasm" => :build
+  depends_on "java" if build.with? "java"
```
Comment (MikeMcQuaid) [inline Formula/jpeg-turbo.rb]: `depends_on :java => :optional`

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

**Comment by MikeMcQuaid** [comment]
@agarie Any news on the comments above? Thanks!

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Pull request closed without merging**